InAppMessaging/Example/Tests/FIRIAMMessageContentDataWithImageURLTests.m

```diff
@@ -81,22 +103,52 @@ - (void)testURLRequestUsingCorrectImageURL {
   XCTAssertEqualObjects([capturedNSURLRequest URL].absoluteString, defaultImageURL);
 }
 
+- (void)testURLRequestUsingCorrectImageURLs {
+  __block NSURLRequest *capturedNSURLRequestForPortraitImage;
+  __block NSURLRequest *capturedNSURLRequestForLandscapeImage;
+  OCMStub([self.mockedNSURLSession
+      dataTaskWithRequest:[OCMArg checkWithBlock:^BOOL(NSURLRequest *request) {
+        if ([request.URL.absoluteString isEqualToString:defaultImageURL]) {
+          capturedNSURLRequestForPortraitImage = request;
+        } else if ([request.URL.absoluteString isEqualToString:defaultLandscapeImageURL]) {
+          capturedNSURLRequestForLandscapeImage = request;
+        }
+        return YES;
+      }]
+        completionHandler:[OCMArg any]  // second parameter is the callback which we don't care in
+                                        // this unit testing
+  ]);
+
+  [_defaultContentDataWithImageURL
+      loadImageDataWithBlock:^(NSData *_Nullable imageData, NSData *_Nullable landscapeImageData,
+                               NSError *error){
```

Comment:
Do we want to do anything in the completion handler here? I guess it's not called?

Reply:
Nope, not called. All important testing magic happens in the method.